prime-router/src/testIntegration/kotlin/transport/RESTTransportIntegrationTests.kt

```diff
@@ -497,7 +497,6 @@ hnm8COa8Kr+bnTqzScpQuOfujHcFEtfcYUGfSS6HusxidwXx+lYi1A==
 
     @Test
     fun `test flexion rest transport`() {
-        val mockHttpClient = mockk<HttpClient>()
```

Comment:
Was this just unused? Seems unrelated to the PR.

Reply:
think so, the linter complained about it